src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_CAPACITY = 16;
+    public static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size = 0;
+    private int threshold;
+    private Node<K, V> [] table;
+
+    class Node<K, V> {
```

Comment:
The inner class Node should be parameterized. This means that instead of `class Node<K, V>`, you should declare it as `class Node`. This is because the outer class `MyHashMap` is already parameterized with `<K, V>`, and these parameters are visible to all of its members, including inner classes.